doc/api/cli.md

```diff
@@ -863,6 +863,22 @@ environment variables.
 
 See `SSL_CERT_DIR` and `SSL_CERT_FILE`.
 
+### `--use-largepages=mode`
+<!-- YAML
+added: REPLACEME
+-->
+
+Re-map the Node.js static code to large memory pages at startup. If supported on
+the target system, this will cause the Node.js static code to be moved onto 2
+MiB pages instead of 4 KiB pages.
+
+The following values are valid for `mode`:
+* `off`: No mapping will be attempted. This is the default.
+* `silent`: If supported by the OS, mapping will be attempted. Failure to map
+  will be ignored and will not be reported.
+* `verbose`: If supported by the OS, mapping will be attempted. Failure to map
```

Comment:
Maybe just `on`? Just to have 'off'/'on' values and avoid possible confusion.

Reply:
I'll do `off`, `on`, and `verbose`, so we can retain the distinction between reporting and silent operation.

Reply:
Maybe then have it `off`, `on`, `silent` as I think most of our features with warnings/errors always print them and rarely allow silent mode so it'd make sense to have it verbose by default.